src/uct/cuda/cuda_copy/cuda_copy_md.h

```diff
@@ -48,6 +49,7 @@ typedef struct uct_cuda_copy_md_config {
     ucs_ternary_auto_value_t    enable_dmabuf;
     ucs_ternary_auto_value_t    enable_fabric;
     uct_cuda_pref_loc_t         pref_loc;
+    ucs_memory_type_t           cuda_async_mem_type;
```

Comment:
Maybe it's worth adding a simple unit test to cover this change?

Reply:
let me try